lib/benchee/suite.ex

```diff
@@ -47,3 +47,93 @@ defimpl DeepMerge.Resolver, for: Benchee.Suite do
     Map.merge(original, override, resolver)
   end
 end
+
+if Code.ensure_loaded?(Table.Reader) do
+  defimpl Table.Reader, for: Benchee.Suite do
```

Comment:
I'd be inclined to move this to its own file just for some separation as it is somehow meaty :)

Reply:
haha I myself implemented the deep_merge resolver here so... maybe not so that it starts breaking the pattern :sweat_smile: